modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -651,7 +651,8 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 /datum/loadout_item/under/miscellaneous/mech_suit
 	name = "Mech Suit"
 	item_path = /obj/item/clothing/under/costume/mech_suit
-	restricted_roles = list(JOB_ROBOTICIST)
+	//restricted_roles = list(JOB_ROBOTICIST)
```

Comment:
Override with `restricted_roles = list()`

Reply:
Tried that, and it didn't work

Reply:
Might be because someone already tried to redefine it in modular_zubbers\code\modules\loadouts\loadout_items\loadout_datum_under.dm

Reply:
Yeah, overriding it with restricted_roles = list() just means no jobs are allowed to have it

Reply:
Then override it as such